### PR TITLE
PRA-366 and PRA-428

### PR DESCRIPTION
**Pull request by aoimasu**
Tutorial component and integration into Nav

### CODE REVIEW

File: cosmos.config.js [hunk 1]
```diff
@@ -3,5 +3,4 @@ module.exports = {
   exclude: [/dist/],
   webpackConfigPath: './config/webpack.dev',
   globalImports: ['./global.css'],
-  port: 9000,
```
Comment (xow) [inline cosmos.config.js]: 😮 how come we are getting rid of this?

Comment (aoimasu) [reply cosmos.config.js]: my mistake, adding it back

**Comment by hen-zone** [comment]
It looks like the final step of the Tutorial renders a `<CheckBox id="neverShowAgain" />`, but this checkbox doesn't do anything. Is this intended?

**Comment by aoimasu** [comment]
When we have the backend/api, that checkbox will be useful for saving tutorial's state for the user

**Comment by BobHayden** [comment]
Does clicking "Got it" need an event handler?

**Comment by hen-zone** [comment]
@aoimasu I see the point of having a checkbox in general, but this particular checkbox does not seem to be wired up to anything, and Tutorial does not expose any way to wire it up, right?

**Pull request merged**